travis/setup_xspec_ds9.sh

```diff
@@ -60,30 +60,27 @@ cd -
 
 ### XSPEC
 xspec_library_path=${xspec_root}/lib/
-xpec_include_path=${xspec_root}/include/
-
-
-# Change build configuration
-sed -i.orig "s/#with-xspec=True/with-xspec=True/g" setup.cfg
-sed -i.orig "s|#xspec_lib_dirs = None|xspec_lib_dirs=${xspec_library_path}|g" setup.cfg
-sed -i.orig "s|#xspec_include_dirs = None|xspec_include_dirs=${xpec_include_path}|g" setup.cfg
-sed -i.orig "s/#cfitsio_libraries/cfitsio_libraries/g" setup.cfg
-sed -i.orig "s/#ccfits_libraries/ccfits_libraries/g" setup.cfg
-sed -i.orig "s/#wcslib_libraries/wcslib_libraries/g" setup.cfg
-sed -i.orig "s|#gfortran_libraries = gfortran|gfortran_libraries= ${libgfortran_name}|g" setup.cfg
+xspec_include_path=${xspec_root}/include/
 
 case "${XSPECVER}" in
-  12.9.1)
-      xspec_version_string="12.9.1"
-      ;;
-  12.9.10e)  # I misspelled the version string, should have been 12.10.0
+  12.10.0e)  
       xspec_version_string="12.10.0"
+      xspec_include_path="$miniconda/envs/build/include"
+      xspec_library_path="$miniconda/envs/build/lib"
       ;;
   *)
-      xspec_version_string="12.9.0"
+      xspec_version_string="12.9.1"
+      sed -i.orig "s/#cfitsio_libraries/cfitsio_libraries/g" setup.cfg
```

Comment:
So, just to confirm that the `*_libraries` settings only need changing in out XSPEC 12.9.1 build?

Reply:
Yes, but that's only true for the xspec binaries we (standalone sherpa and ciao) compile. With a default heasarc installation you'd still need to link the libraries heasarc didn't link explicitly, so you'd need to set `*_libraries` and cross your fingers.

Reply:
Yeah - I was just clarifying that the change was intentional for our setup only ;-)